### PR TITLE
chore: upgrade cypress to v13.6.6

### PR DESCRIPTION
**Pull request by kaylachun**
# Elements Default PR Template
Upgrades `cypress` to latest version: `v13.6.6`
In general, make sure you have: (check the boxes to acknowledge you've followed this template)

- [ ] Read [`CONTRIBUTING.md`](../CONTRIBUTING.md)

#### Other Available PR Templates:

- Release: https://github.com/stoplightio/elements/compare?template=release.md
  - [ ] [Read the release section of `CONTRIBUTING.md`](../CONTRIBUTING.md#releasing-elements)

### CODE REVIEW

File: cypress.config.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  screenshotsFolder: 'cypress/results/screenshots',
+  videosFolder: 'cypress/results/videos',
+  e2e: {
+    // We've imported your old cypress plugins here.
+    // You may want to clean this up later by importing these.
+    setupNodeEvents(on, config) {
+      return require('./cypress/plugins/index.js')(on, config);
+    },
+    baseUrl: 'http://localhost:4200/',
+    defaultCommandTimeout: 10000,
```
Comment (kaylachun) [inline cypress.config.ts:18]: Had to increase timeout to 10s to get the angular e2e tests to pass..

File: cypress.config.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  reporter: 'junit',
+  reporterOptions: {
+    mochaFile: 'cypress/results/output.xml',
+    toConsole: false,
+  },
+  screenshotsFolder: 'cypress/results/screenshots',
+  videosFolder: 'cypress/results/videos',
+  e2e: {
+    // We've imported your old cypress plugins here.
+    // You may want to clean this up later by importing these.
```
Comment (mallachari) [inline cypress.config.ts:13]: Should we remove this comment?

File: cypress.config.ts (hunk 1 above)
Comment (mallachari) [reply cypress.config.ts:18]: I think it's acceptable

**Review by mallachari — APPROVED** [review]

**Comment by kaylachun** [comment]
@mallachari After doing some testing, it seems like upgrading `testing-library/cypress` caused the long test times.  I'm going to only upgrade cypress to the latest version since that fixes the security issue and the tests can run within the default timeout.

**Pull request merged**